numpy/lib/tests/test_io.py

```diff
@@ -2268,6 +2268,7 @@ def test_save_load_memmap(self):
             np.save(path, a)
             data = np.load(path, mmap_mode='r')
             assert_array_equal(data, a)
+            del data
```

Comment:
What's the rationale for this line? A comment might be nice

Reply:
This is required such that numpy closes the mem-mapped file correctly. Otherwise you will get weird errors on Windows.